packages/api/app/distribution-routes.js

```diff
@@ -90,7 +90,38 @@ async function ensureAuthorizedOrRedirect(req, res, next) {
   return next();
 }
 
+// TODO implement each method with CSDAP authentication
```

Comment:
I'm a little unsure if this is correct place to put these. This file was originally just figuring out if they're authorized and sending them to EDLogin. Do we want to make a new file for a place for this stuff, or are we going to change the purpose of this file

Reply:
It's currently not used except for local distribution api (not sure who uses it), and we can make local distribution api to work for cognito (since auth part is ignored).

Reply:
Okay, I guess I'm fine with that if that's going to be the plan. We might decided to reorganize later anyway